test/integration/legacy_routes/handler_test.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+require 'pry'
+require 'test_helper'
+
+class LegacyRoutes::UrlTest < ActionDispatch::IntegrationTest
+  include IntegrationTest::DefaultHelper
+
+  test 'redirects to groups with without groups in url' do
+    account_url = "/teams/#{teams(:team1).id}/groups/#{groups(:group1).id}"
```

Comment:
this is not a account_url, is it ?
name this legacy_account_url ... this makes your test much more readable